src/main.js

```diff
@@ -9,70 +8,94 @@ var btnMakeNewCover = document.querySelector(".make-new-button");
 var homeView = document.querySelector(".home-view");
 var savedView = document.querySelector(".saved-view");
 var formView = document.querySelector(".form-view");
-var inputCover = document.querySelector(".user-cover");///vars below will be on Jessica's branch
+var inputCover = document.querySelector(".user-cover")
 var inputTitle = document.querySelector(".user-title");
 var inputDescriptor1 = document.querySelector(".user-desc1");
 var inputDescriptor2 = document.querySelector(".user-desc2");
 var form = document.querySelector("form");
 
 var savedCovers = [
-  new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
+  new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
 ];
 
+// Event listeners
 window.addEventListener("load", createRandomCover);
 btnMakeNewCover.addEventListener("click", showForm);
 btnRandomCover.addEventListener("click", createRandomCover);
 btnViewSavedCover.addEventListener("click", showSaved);
 btnHome.addEventListener("click", showHome);
+btnMakeMyBook.addEventListener("click", createNewCover);
 form.addEventListener('submit', handleForm);
 
+//Prevents page reload upon submitting form
 function handleForm(event) { event.preventDefault(); }
 
+//Hides home view and associated buttons, displays home button
 function leaveHome() {
-  homeView.classList.add("hidden");
-  btnSaveCover.classList.add("hidden");
-  btnRandomCover.classList.add("hidden");
+  homeView.classList.add("hidden");
+  btnSaveCover.classList.add("hidden");
+  btnRandomCover.classList.add("hidden");
   btnHome.classList.remove("hidden");
 }
-
+//Displays home view and associated buttons, hides home button
 function returnHome() {
   homeView.classList.remove("hidden");
   btnSaveCover.classList.remove("hidden");
   btnRandomCover.classList.remove("hidden");
   btnHome.classList.add("hidden");
+  formView.classList.add("hidden");
 }
 
+// Retrieves random array index
 function getRandomIndex(array) {
-  return Math.floor(Math.random() * array.length);
+  return Math.floor(Math.random() * array.length);
 }
-//When user hits show new random button, a new cover is displayed
+
+//Event handler for btnRandomCover
 function createRandomCover() {
-  var randomCover = new Cover(covers[getRandomIndex(covers)], titles[getRandomIndex(titles)], descriptors[getRandomIndex(descriptors)], descriptors[getRandomIndex(descriptors)])
-  coverImage.src = randomCover.cover;
-  title.textContent = randomCover.title;
-  descriptor1.textContent = randomCover.tagline1;
-  descriptor2.textContent = randomCover.tagline2;
+  var randomCover = new Cover(covers[getRandomIndex(covers)], titles[getRandomIndex(titles)], descriptors[getRandomIndex(descriptors)], descriptors[getRandomIndex(descriptors)])
```

Comment:
similar to one of my previous comments, i'd break all the inputs into variables and pass the variables in as arguments. it'll make the function a lot more readable.

Reply:
Would you suggest declaring these variables in the function? This way we avoid having too many global variables?

Reply:
yes definitely, i'd just declare them right above the instance!

Reply:
Thank you!